doc/workflows/automated_testing.rst

```diff
@@ -105,19 +105,35 @@ That would represent an ``old`` test session with 20 iterations of the test, 15
 of which passed and 5 failed. The ``new`` session would have had 100
 iterations, out of which 80 passed and 20 failed.
 
+.. note:: This kind of experiments only fixes some marginal totals and
```

Comment:
Nit: might want a line break or two in there to avoid having a Big Scary Wall of Text.

Reply:
It's now a big scary wall of statistics gibberish :-)